home.js

```diff
@@ -10,4 +10,195 @@ window.onload = function () {
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Website Protfolio',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Profesional Art',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Data Dashboard',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Website Protfolio',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+];
+
+function showPopup() {
+  document.querySelector('.multi-cont').style.display = 'flex';
+}
+
+/* ----------------- Building Popup (Mobile) ---------------- */
+document.addEventListener('DOMContentLoaded', (event) => {
+  const popupSection = document.querySelector('.multi-cont');
+  const popupCont = document.createElement('div');
+  popupCont.className = 'multi-cont2';
+  popupSection.append(popupCont);
+  const closeSign = document.createElement('img');
+  closeSign.classList.add('closesign');
+  closeSign.src = './assets/Icon.png';
+  closeSign.alt = 'closesign';
+  closeSign.onclick = hidePopup;
+  const popupTitle = document.createElement('h3');
+  popupTitle.className = 'multi-stories';
+  const popupUl = document.createElement('ul');
+  popupUl.className = 'ul-tech1';
+  const popupImg = document.createElement('img');
+  popupImg.alt = 'project';
+  popupImg.className = 'image1';
+  const popupDescription = document.createElement('p');
+  popupDescription.className = 'multitext';
+  const liveButton = document.createElement('button');
+  liveButton.innerText = 'See Live';
+  liveButton.className = 'btn-pop1 hover';
+  const sourceButton = document.createElement('button');
+  sourceButton.innerText = 'See Source';
+  sourceButton.className = 'btn-pop2 hover';
+  popupCont.append(
+    closeSign,
+    popupTitle,
+    popupUl,
+    popupImg,
+    popupDescription,
+    liveButton,
+    sourceButton,
+  );
```

Comment:
As requested by the previous reviewer:
- [ ] kindly use the correct `background-color` for the buttons (`#FF6B00`)
- [ ] kindly add the icons to the buttons as shown in Figma

[ Optional ]

- [ ] Good job on making the buttons in a single row, however one of them is lower than the other, please try to make them aligned on the same level 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/202759973-f22ef593-ae49-4f98-a888-53038e54151f.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/202760014-dd2b0805-544f-4982-976a-48f91bc0ccbf.png)